addons/purchase/migrations/11.0.1.2/noupdate_changes.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="email_template_edi_purchase" model="mail.template">
+    <field name="body_html"><![CDATA[
```

Comment:
If you choose to refresh the body of a particular email template, please also remove its translations.

Reply:
You are right. Done. :)